react-responsive-modal/src/index.tsx

```diff
@@ -69,6 +70,12 @@ export interface ModalProps {
    * Default to true.
    */
   focusTrapped?: boolean;
+  /**
+   * Options for focus trapping.
+   *
+   * Default to undefined.
+   */
+  focusTrapOptions?: FocusTrapOptions;
```

Comment:
as we have one option what do you think about changing the name to something like `openFocusRef` or another better name if you have a good idea? This would also simplify the API

Reply:
I'm thinking a bit about ability to expand options and keep the API backwards compatible. I also think that `focusTrapOptions` is way more explicit and tells you about what you can do. I believe that's important from DX point of view. I also have no idea how to name it better :wink:

Reply:
I agree that it is more explicit, but the name of the prop can also make it really explicit. I remember doing something like this on one of my packages and in the end I decided to go against that way. It's hard to document properly these options in the documentation (in a table) and users tend to prefer the root props too (from my experience).

Reply:
I'll try to figure something out then - let me know if you have any ideas :+1:

Reply:
Chakra-ui is using `initialFocusRef`, the name is quite descriptive https://chakra-ui.com/docs/overlay/modal#props

Reply:
That sounds really good. I'm adding changes and pushing it to the branch :+1: